contracts/src/tests/test_ownable.cairo

```diff
@@ -121,3 +122,45 @@ fn test_renounce_ownership_panics_if_not_owner() {
     set_caller_address(other_user);
     Ownable::renounce_ownership();
 }
+//
+// General ownable contract tests
+//
+
+#[abi]
+trait IOwnable {
+    fn owner() -> ContractAddress;
+    fn proposed_owner() -> ContractAddress;
+    fn transfer_ownership(new_owner: ContractAddress);
+    fn accept_ownership();
+    fn renounce_ownership();
+}
```

Comment:
We could declare this inside the ownable module so it can be imported elsewhere?

Reply:
I copied it over but it doesnt seem to work in this test file when importing from Ownable 🤔

Reply:
tried that but still doesnt work (methods not found on type IOwnableDispatcher)